README.tfdoc.hcl

```
@@ -575,6 +601,38 @@ section {
           `RAPID`, `REGULAR` and `STABLE`.
         END
       }
+      variable "node_pool_auto_config" {
+        type        = any
```

please never reference `any` in docs... 

```suggestion
        type        = object(node_pool_auto_config)
```